presto-docs/src/main/sphinx/language/reserved.rst

```diff
@@ -50,6 +50,9 @@ Keyword                        SQL:2016      SQL-92
 ``INTO``                       reserved      reserved
 ``IS``                         reserved      reserved
 ``JOIN``                       reserved      reserved
+``JSON_EXISTS``                reserved
```

Comment:
Would you add documentation for these new features?

Reply:
Document is in #18256.